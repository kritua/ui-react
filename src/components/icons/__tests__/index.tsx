import React from 'react';
import renderer from 'react-test-renderer';

import { SVG } from './../index';

it('Default', () => {
    const tree = renderer.create(<SVG />).toJSON();

    expect(tree).toMatchSnapshot();
});

it('ClassName string', () => {
    const tree = renderer.create(<SVG className="my-class" />).toJSON();

    expect(tree).toMatchSnapshot();
});
