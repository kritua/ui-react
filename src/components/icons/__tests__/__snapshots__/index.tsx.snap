// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClassName string 1`] = `
<svg
  className="my-class"
  height={24}
  viewBox="0 0 24 24"
  width={24}
  xmlns="http://www.w3.org/2000/svg"
/>
`;

exports[`Default 1`] = `
<svg
  height={24}
  viewBox="0 0 24 24"
  width={24}
  xmlns="http://www.w3.org/2000/svg"
/>
`;
